bin/eslint.js

```diff
@@ -16,7 +18,13 @@ if (debugMode) {
 debug("START", process.argv)
 debug("ROOT", cwd)
 
-const binPath = require("../lib/get-local-eslint")(cwd) || require("../lib/get-bin-eslint-js")(cwd)
+const yarnGlobalDir = spawnSync("yarn", ["global", "dir"])
```

Comment:
emm, I'm not sure about this -- yarn isn't an official package manager that ships with Node.

Reply:
if user do not install yarn, eslint-cli do not fail.
Still no good?

Reply:
there was a similar situation in eslint core: https://github.com/eslint/eslint/issues/9863 , though I'm not sure this one can be exception.

Reply:
I think we should not need to depend on `yarn global` and `npm root -g` commands. If both `eslint-cli` and `eslint` are installed into global, `require("eslint")` should load the global-installed eslint.